src/components/HomeAnimation.vue

```diff
@@ -17,20 +17,24 @@
         </div>
 
         <div class="c-main-wrapper" id="header1">
-          <!-- <div class="is-posttitle">GROUP INCOME</div>
-          <h1 class="is-title-1 c-main-title">
+          <h1 class="is-font-poppin c-page-title">Group Income</h1>
+          <!-- <h1 class="is-title-1 c-main-title">
             Basic Income for your
             friends and family.
           </h1> -->
-          <div class="video-wrapper">
-            <!-- <iframe
-              id="odysee-iframe"
-              style="width:100%; aspect-ratio:16 / 9;"
-              src="https://odysee.com/%24/embed/%40okTurtles%3Ae%2FokTurtles-Group-Income-Launch-Special-Announcement%3A3?r=J91Yd9UJDqP36LmGQZaGwG95T1NcnzDW"
-              allowfullscreen>
-            </iframe> -->
-            <iframe style="width:100%; aspect-ratio:16 / 9;" src="https://www.youtube-nocookie.com/embed/md54gZAtWA8?si=QH8HqnBwWWHVeKOf" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+          <div class="c-video-wrapper">
+            <!-- <VideoPlayer class="has-deep-box-shadow"
+              src='https://okturtles.org/videos/Group_Income_1.0_Launch_Special_Announcement.mp4'
```

Comment:
Note that this video has been deleted, so for testing you can update the URL to

https://okturtles.org/videos/group_income_ending_big_tech_control.mp4